cypress/integration/consumersCart.spec.js

```diff
@@ -0,0 +1,17 @@
+describe('consumersCart', function () {
+
+it('Add one item to the cart', () => {
+
+  cy.visit('/')
+    cy.get('[data-test="username"]').click()
+      .type('standard_user')
+    cy.get('[data-test="password"]').click()
+      .type('secret_sauce')
+    cy.get('[data-test="login-button"]').click()
+    
+  cy.get('#add-to-cart-sauce-labs-backpack').click()
+  cy.get('#shopping_cart_container').click()
+  cy.get('#item_4_title_link').should('include.text', 'Sauce Labs Backpack')
```

Comment:
Optional: 
Right now you are checking that that the name of the #item_4 includes some text.
It will be better to get the list of items from the cart (class="cart_list"), and make sure that #item_4 (class="cart_item") is one of them.

Reply:
Thanks! I  guess I can do it a bit later